packages/runner/lib/exec.ts

```diff
@@ -47,7 +46,7 @@ export async function exec(
 
     const filename = `${nangoProps.syncConfig.sync_name}-${nangoProps.providerConfigKey}.js`;
 
-    return await tracer.trace<Promise<RunnerOutput>>(SpanTypes.RUNNER_EXEC, async (span) => {
+    return await tracer.trace<Promise<RunnerOutput>>('nango.runner.exec', async (span) => {
```

Comment:
why this change?

Reply:
After removing log, there was only 2 spans defined in telemetry file, it seems we are not using this pattern so to be consistent I removed it